xbmc/storage/cdioSupport.cpp

```diff
@@ -72,39 +70,12 @@ signature_t CCdIoSupport::sigs[] =
 #undef DEBUG_CDIO
 
 static void
-xbox_cdio_log_handler (cdio_log_level_t level, const char message[])
-{
-#ifdef DEBUG_CDIO
-  switch (level)
-  {
-  case CDIO_LOG_ERROR:
-    CLog::Log(LOGDEBUG,"**ERROR: %s", message);
-    break;
-  case CDIO_LOG_DEBUG:
-    CLog::Log(LOGDEBUG,"--DEBUG: %s", message);
-    break;
-  case CDIO_LOG_WARN:
-    CLog::Log(LOGDEBUG,"++ WARN: %s", message);
-    break;
-  case CDIO_LOG_INFO:
-    CLog::Log(LOGDEBUG,"   INFO: %s", message);
-    break;
-  case CDIO_LOG_ASSERT:
-    CLog::Log(LOGDEBUG,"!ASSERT: %s", message);
-    break;
-  default:
-    //cdio_assert_not_reached ();
-    break;
-  }
-#endif
-}
 
 //////////////////////////////////////////////////////////////////////
 // Construction/Destruction
 //////////////////////////////////////////////////////////////////////
 CLibcdio::CLibcdio(): s_defaultDevice(NULL)
 {
-  cdio_log_set_handler( xbox_cdio_log_handler );
```

Comment:
You shoulnd't remove the log handler at all. Just change the name to something that doesn't contain `xbox` because it's still needed to see what's going on inside `cdio`.